osu.Game/Screens/Select/FooterButton.cs

```diff
@@ -55,6 +55,8 @@ public Color4 SelectedColour
         private readonly Box box;
         private readonly Box light;
 
+        public override bool ReceiveMouseInputAt(Vector2 screenSpacePos) => box.ReceiveMouseInputAt(screenSpacePos);
```

Comment:
Instead of this, how about moving the shear to the footer button itself?

Reply:
If you mean this, then it'll result with:

![](https://puu.sh/AQsMe/48b18ab75e.png)

```
        public FooterButton()
        {
            Shear = shearing;

            Children = new Drawable[]
            {
                box = new Box
                {
                    RelativeSizeAxes = Axes.Both,
                    EdgeSmoothness = new Vector2(2, 0),
                    Colour = Color4.White,
                    Alpha = 0,
                },
                light = new Box
                {
                    Height = 4,
                    EdgeSmoothness = new Vector2(2, 0),
                    RelativeSizeAxes = Axes.X,
                },
                spriteText = new OsuSpriteText
                {
                    Anchor = Anchor.Centre,
                    Origin = Anchor.Centre,
                }
            };
        }
```